r10k.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.license  = 'Apache-2.0'
 
   s.add_dependency 'colored2',   '3.1.2'
-  s.add_dependency 'cri',       '2.15.6'
+  s.add_dependency 'cri',       '>= 2.15.6'
```

Comment:
We might want to do `'~> 2.15'` or  `['>= 2.15.6', '< 3.0.0']` if we're worried about maintaining support for certain Ruby versions (major version bumps have bitten us in the past).

Reply:
updated it :)